andela/trime
# Feature/project not found

#### What does this PR do?

Renders a `ProjectNotFound` component instead of the `ProjectPage` if the project was not found.

#### Description of Task to be completed?

Implement a custom 404 page with a message describing the error when a user navigates to an invalid project

#### How should this be manually tested?

Browse to an non-existent project e.g http://localhost:3000/orgs/1/projects/100000000000000. You should see a custom 404 page.

#### What are the relevant pivotal tracker stories?

[Feature #120097095] (https://www.pivotaltracker.com/projects/1454736/stories/120097095)


<img width="1680" alt="screen shot 2016-06-28 at 10 16 29 am" src="https://cloud.githubusercontent.com/assets/17295379/16407337/db968504-3d1b-11e6-8029-3563f163a236.png">
<img width="1680" alt="screen shot 2016-06-28 at 10 25 47 am" src="https://cloud.githubusercontent.com/assets/17295379/16407342/e772f114-3d1b-11e6-987c-704562318481.png">
<img width="1680" alt="screen shot 2016-06-28 at 10 14 21 am" src="https://cloud.githubusercontent.com/assets/17295379/16407355/f36a9788-3d1b-11e6-9d6f-96299351d5ce.png">
<img width="1680" alt="screen shot 2016-06-28 at 10 43 36 am" src="https://cloud.githubusercontent.com/assets/17295379/16407603/391c4bc2-3d1d-11e6-91ca-f48931036b7c.png">

@andela-ladenusi
